modules/seqexec/server/src/main/scala/seqexec/server/keywords/GDSClient.scala

```diff
@@ -33,22 +34,19 @@ final case class GDSClient(client: Client[IO], gdsUri: Uri)
             <string>{id}</string>
           </value>
         </param>
-        <param>
-          <value>
-            <array>
-              <data>
-                {
-                  ks.keywords.map { k =>
-                    <value><string>{s"${k.name},${KeywordType.gdsKeywordType(k.keywordType)},${k.value}"}</string></value>
-                  }
-                }
-              </data>
-            </array>
-          </value>
-        </param>
+        {keywordsParam(ks)}
       </params>
     </methodCall>
 
+  private def handleConnectionError
+    : PartialFunction[Throwable, SeqexecFailure] = {
```

Comment:
Why is this a `PartialFunction`?

Reply:
We pass it to `EitherT.leftMap` which takes that signature

Reply:
It does? Weird.

Reply:
https://github.com/typelevel/cats/blob/master/core/src/main/scala/cats/data/EitherT.scala#L109

?

Reply:
Oh, i got confused with the transformers i guess when trying to disentangle the monad transformers stack

Reply:
I would 👍 a PR to remove all the type aliases ;-)

Reply:
yeah, maybe I can come up with some magic sed command to do it

Reply:
Fixed the signature